addons/website/models/website_menu.py

```diff
@@ -67,12 +83,19 @@ def create(self, vals):
                     'parent_id': website.menu_id.id,
                 })
                 res = super(Menu, self).create(w_vals)
+                res._remove_domain()
             # if creating a default menu, we should also save it as such
             default_menu = self.env.ref('website.main_menu', raise_if_not_found=False)
             if default_menu and vals.get('parent_id') == default_menu.id:
                 res = super(Menu, self).create(vals)
+                res._remove_domain()
         return res  # Only one record is returned but multiple could have been created
 
+    def write(self, vals):
+        result = super().write(vals)
+        result._remove_domain()
```

Comment:
result._remove_domain() should be self._remove_domain()

what's the reason to call super() instead of super(Menu, self) ?

Reply:
> what's the reason to call super() instead of super(Menu, self) ?

I'd say the opposite question... Why not? This is Python 3 already, both calls are equivalent but `super()` is more comfortable.